src/viewport.js

```diff
@@ -514,7 +514,7 @@ $.Viewport.prototype = {
      * @param {OpenSeadragon.Rect} bounds
      * @returns {OpenSeadragon.Rect} constrained bounds.
      */
-    _applyBoundaryConstraints: function(bounds) {
+     _applyBoundaryConstraints: function(bounds) {
```

Comment:
This is a super tiny nitpick, but it seems this line got a space added at the beginning.